### ISSUE TITLE
Can you release model zoo configs?

### ISSUE THREAD

**Issue by moyans** [KaiyangZhou/deep-person-reid#254]

**Comment by KaiyangZhou** [comment]
most of them are in the configs/ folder

otherwise you can figure out the training params from the file names

**Comment by moyans** [comment]
@KaiyangZhou  Thank you for your reply. About Mark1501, if I use SGD to optimize, choose what initial learning rate is better? If I multiply the batch size, Does it mean that I can multiply the learning rate?

**Comment by KaiyangZhou** [comment]
the lr depends on the batch size, check this out https://arxiv.org/abs/1706.02677
